source/cipher.c

```diff
@@ -928,6 +928,9 @@ int aws_cryptosdk_rsa_encrypt(
     const struct aws_byte_cursor plain,
     const struct aws_string *rsa_public_key_pem,
     enum aws_cryptosdk_rsa_padding_mode rsa_padding_mode) {
+    AWS_PRECONDITION(aws_byte_buf_is_valid(cipher));
+    AWS_PRECONDITION(aws_byte_cursor_is_valid(&plain));
+    AWS_PRECONDITION(rsa_public_key_pem != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Updated!